pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[tool.poetry]
+name = "deep-learning-playground"
+version = "0.1.0"
+description = "Web Application where people new to Deep Learning can input a dataset and toy around with basic Pytorch modules without writing any code"
+authors = ["DSGT-DLP <dlp@datasciencegt.org>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "deep_learning_playground"}]
+
+[tool.poetry.dependencies]
+python = ">=3.9, <3.10"
+flask = "2.2.3"
+flask-cors = "*"
+flask-socketio = "*"
+numpy = "*"
+pandas = "*"
+matplotlib = "*"
+seaborn = "*"
+mypy = "*"
+pytest = "*"
+scikit-learn = "*"
+scipy = "1.7.3"
+boto3 = "*"
+pyyaml = "*"
+statsmodels = "*"
+torch = "*"
+transformers = "*"
+pre-commit = "*"
+torchvision = "*"
+timm = "*"
+wwf = "*"
+fastai = "==2.7.4"
+firebase_admin = "*"
+pyrebase4 = "*"
+moto = "*"
+docker = "*"
+ggshield = "*"
+black = "^23.3.0"
+psutil = "^5.9.5"
+logger = "^1.4"
+
+[tool.poetry.dev-dependencies]
+python = ">=3.9"
+flake8 = "*"
```

Comment:
we should make it clear that if you are developing, you should also have thr dev dependencies installed on top of the prod deps

Reply:
I see. so this means updating the ReadME file to provide clear instructions on installing both production and development dependencies?

maybe something like:

-  Install production dependencies: `poetry install --no-dev`
- Install development dependencies: `poetry install`

By installing both production and development dependencies, you will have all the necessary packages to run the application and also the additional tools for testing, linting, and code formatting.

Reply:
yup. so poetry install = dev + prod deps? @codingwithsurya

Reply:
> . so poetry install = dev + prod deps?

yea exactly